themes-default/slim/.babelrc

```diff
@@ -1,8 +1,15 @@
 {
   "presets": [
-    "env"
+    "@babel/preset-env"
   ],
   "plugins": [
-    "transform-object-rest-spread"
-  ]
+    "@babel/plugin-proposal-object-rest-spread"
+  ],
+  "env": {
+    "test": {
+      "plugins": [
+        "istanbul"
```

Comment:
What's this needed for?

Reply:
1. I followed https://istanbul.js.org/docs/tutorials/es2015/ and it doesn't collect any coverage without it.
2. You added the dependency in https://github.com/pymedusa/Medusa/pull/4527/commits/0c7e563ab1a59832c91fcdc979e26a82ada83499, but it wasn't being used.
    > A babel plugin that adds istanbul instrumentation to ES6 code.